File: backend/src/main/java/com/tourneyhub/backend/dto/tournament/SimpleTournamentDto.java
package com.tourneyhub.backend.dto.tournament;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class SimpleTournamentDto {

    private Long id;

    private String name;

    private String description;

    private String banner;

    private Integer keyCount;

    private Integer minTeamSize;

    private Integer maxTeamSize;

    private Integer minPlayerRank;

    private Integer maxPlayerRank;

    private boolean concluded;

    private boolean published;
}

File: tourneyhub/src/dto/tournament/SimpleTournamentDto.ts
export interface SimpleTournamentDto {
    id: string,
    name: string,
    description: string,
    banner: string,
    keyCount: number,
    minTeamSize: number,
    maxTeamSize: number,
    minPlayerRank: number,
    maxPlayerRank: number,
    concluded: boolean,
    published: boolean
}